bin/ch/WScriptJsrt.cpp

```diff
@@ -725,7 +725,7 @@ bool WScriptJsrt::InstallObjectsOnObject(JsValueRef object, const char* name, Js
     memcpy(path, msg, (size_t)str_len);          \
     path[str_len] = char(0)
 
-void GetBinaryLocation(char *path, const uint32_t size)
+void GetBinaryLocation(char *path, const unsigned size)
```

Comment:
@dilijev  what do you mean by `Since the value passed in is a size_t` ? You mean `sizeof` returns `size_t` ?

Reminder, that param is used only for max buffer size that is actually filled in by a system call.